tlscanary/report.py

```diff
@@ -2,15 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import cert
```

Comment:
This was *precisely* where it was supposed to be: in the second include block that bundles all the tlscanary modules. Please move it back again.